types/apostrophe/index.d.ts

```diff
@@ -0,0 +1,302 @@
+// Type definitions for apostrophe 2.67
+// Project: https://github.com/apostrophecms/apostrophe#readme
+// Definitions by: Aaron Holderman <https://github.com/afholderman>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+export = apostrophe;
+export as namespace apos;
+
+declare function apostrophe(options: any, ...args: any[]): any;
+
+declare namespace apostrophe {
+    const moogBundle: {
+        directory: string;
+        modules: string[];
+    };
+
+    const prototype: {};
```

Comment:
Out of curiosity, why would a namespace have a `prototype` property?

Reply:
That came in as part of the defs generated by dts-gen. Removing it did not seem to effect anything so I'll push that change.